src/clarity-angular/wizard/providers/wizard-navigation.ts

```diff
@@ -58,11 +58,10 @@ export class WizardNavigationService implements OnDestroy {
             } else {
                 this.cancel();
             }
-            // SPECME
         });
 
         this.pagesResetSubscription = this.pageCollection.pagesReset.subscribe(() => {
-            this.setLastEnabledPageCurrent();
+            this.setFirstPageCurrent();
```

Comment:
Sorry I am new to the wizard so don't know much but this looks like a behavior change. Does this mean that instead of the last enabled page, the first page of the wizard will be the current page when the reset is called?